.gitignore

```diff
@@ -30,6 +30,8 @@ build/Release
 node_modules
 jspm_packages
 .hintrc
+package-lock.json
+yarn.lock
```

Comment:
yarn.lock should not be added to the gitignore, we use yarn so adding package-lock is unnecessary. Please revert both of these

Reply:
so true

Reply:
fixed